src/components/post/index.js

```diff
@@ -4,33 +4,39 @@ import Comment from "../comment"
 import OptionsIcon from "../optionsIcon"
 import EditPostModal from "../editPostModal"
 import ProfileCircle from "../profileCircle"
-import { useEffect, useState } from "react"
+import { useState, useEffect, useContext } from "react"
+import { toggleLike } from "../../service/apiClient"
 import "./style.css"
 
 import emptyHeart from "../../assets/icons/empty-heart.png"
 import heart from "../../assets/icons/heart.png"
 import emptyComment from "../../assets/icons/empty-comment.png"
 import comment from "../../assets/icons/comment.png"
+import jwtDecode from "jwt-decode"
```

Comment:
Should we still be importing `jwtDecode`?

Reply:
Yes because I do jot need only token I need userId